tests/album-update.test.js

```diff
@@ -0,0 +1,49 @@
+const { expect } = require('chai');

```

Comment:
This test doesn't run.

Reply:
First you need the artists, albums and album. In a beforeEach then you create an artist so you can use its id and then create the album entries.